tensorflow/python/data/util/random_seed_test.py

```diff
@@ -18,43 +18,98 @@
 from __future__ import division
 from __future__ import print_function
 
+from absl.testing import parameterized
+
 from tensorflow.python.data.util import random_seed as data_random_seed
 from tensorflow.python.eager import context
+from tensorflow.python.data.kernel_tests import test_base
+from tensorflow.python.framework import combinations
 from tensorflow.python.framework import constant_op
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import random_seed
-from tensorflow.python.framework import test_util
 from tensorflow.python.platform import test
 
 
-class RandomSeedTest(test.TestCase):
+# NOTE(vikoth18): Arguments of parameterized tests are lifted into lambdas to make
+# sure they are not executed before the (eager- or graph-mode) test environment
+# has been set up.
+#
+class RandomSeedTest(test_base.DatasetTestBase, parameterized.TestCase):
+
+  @combinations.generate(
+      combinations.times(
+          test_base.default_test_combinations(),
+          combinations.combine(test_case_fn=[
```

Comment:
move the definition of the test cases into a helper function to avoid the excessive indentation ... see https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/data/experimental/kernel_tests/optimization/filter_fusion_test.py for an example

Reply:
This comment applies throughout this PR.

Reply:
addressed in the commit